packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -438,6 +444,7 @@ class _DismissibleState extends State<Dismissible> with TickerProviderStateMixin
           direction: _dismissDirection,
           reached: _dismissThresholdReached,
           previousReached: oldDismissThresholdReached,
+          progress: _moveController!.value,
```

Comment:
Should this be the `_moveController!.value` or the progress towards the actual threshold?

Reply:
If I understand correctly, the threshold is the value after which the dismiss action completes sucessfully if the pointer is lifted?

The value we want to report in the callback is the actual visibility of the dismissible, regardless of the threshold, since it will be used to synchronize other elements to what the dismissible is doing on screen, e.g. fading the dismissible as it's dragged offscreen - you'd want the opacity to be at 0.0 just as the widget is completely offscreen.

Reply:
> The value we want to report in the callback is the actual visibility of the dismissible, regardless of the threshold, since it will be used to synchronize other elements to what the dismissible is doing on screen, e.g. fading the dismissible as it's dragged offscreen - you'd want the opacity to be at 0.0 just as the widget is completely offscreen.

Something like this would be excellent to include in the documentation, up around line 250ish :)

Reply:
Thanks. I've tried to improve it.
As a software developer, I'm chronically incapable of writing good concise documentation so feedback is welcome :-)